dotCMS/src/integration-test/java/com/dotcms/rendering/velocity/services/HTMLPageAssetRenderedTest.java

```diff
@@ -1132,6 +1125,89 @@ public void shouldReturnParserContainerUUID() throws Exception {
         Assert.assertTrue(html.matches(regexExpected));
     }
 
+    private HttpSession getHttpSession(HttpServletRequest mockRequest) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'mockRequest' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5041322)